app/src/pages/inside/historyPage/historyView/historyTable/historyItemsGrid/historyItem/messageBadge/messageBadge.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { withTooltip } from 'components/main/tooltips/tooltip';
+import Parser from 'html-react-parser';
+import styles from './messageBadge.scss';
+
+const cx = classNames.bind(styles);
+
+const MessageBadgeToolTip = ({ data }) => (
+  <div className={cx('tooltip-content')}>
+    {data.map((item) => (
+      <div className={cx('content-container')} key={item.ticketId.length}>
```

Comment:
Strange property provided to `key`. Is length of `ticketId` really unique in array of `data`?